chevah/brink
# [#3863] Add an non overwrite option to BrinkFilesystem.copyFolder.

Scope
=====

BrinkFilesystem?.copyFolder should have an option for copying without overwriting files. 


Changes
=======

Added an `overwrite` argument to the `copyFolder` method, it defaults to `True` keeping the old behavior.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please check if changes make sense.
